libfwupdplugin/fu-mei-device.c

```diff
@@ -65,11 +67,25 @@ fu_mei_device_to_string(FuDevice *device, guint idt, GString *str)
 static gboolean
 fu_mei_device_probe(FuDevice *device, GError **error)
 {
-	/* only open the main device */
-	if (fu_udev_device_get_device_file(FU_UDEV_DEVICE(device)) == NULL) {
-		g_set_error_literal(error, FWUPD_ERROR, FWUPD_ERROR_NOT_SUPPORTED, "no device");
+	FuMeiDevice *self = FU_MEI_DEVICE(device);
+	FuMeiDevicePrivate *priv = GET_PRIVATE(self);
+	const gchar *uuid;
+
+	/* the kernel is missing `dev` on mei_me children */
```

Comment:
is this a kernel bug that should be filed somewhere?

Reply:
I'm not sure; I can certainly file one if you think it's worthwhile.

Reply:
I don't feel strongly on it.